typescript/core/src/pre0/parse.ts

```diff
@@ -191,3 +189,14 @@ export function parseRecord(
       throw new Error("Not yet implemented");
   }
 }
+
+function isChannelInfoEqual(a: ChannelInfo, b: ChannelInfo): boolean {
```

Comment:
Similar thoughts to the channel info comparison below - maybe a crc on the bytes would be effective and work if we ever add a new field.